doc/dev/documenting.rst

```diff
@@ -2,6 +2,20 @@
  Documenting Ceph
 ==================
 
+User documentation
+==================
+
+The documentation on docs.ceph.com is generated from the restructuredText
+sources in ``/doc/`` in the Ceph git repository.
```

Comment:
I've added a para mentioning admin/build-doc and github's "View" (which was new to me!)